docker/build_jar.sh

```diff
@@ -5,11 +5,8 @@
 # chmod +x must be set for this script
 set -e
 
-# Get the last commit hash and set it as an environment variable
-GIT_HASH=$(git log -n 1 --pretty=format:%h)
-
 # make jar.  cache sbt dependencies. capture output and stop db before returning.
-docker run --rm -v $PWD:/working -v sbt-cache:/root/.sbt -v jar-cache:/root/.ivy2 -v coursier-cache:/root/.cache/coursier hseeberger/scala-sbt:eclipse-temurin-17.0.2_1.6.2_2.13.8 /working/docker/clean_install.sh /working
+docker run --rm -e DOCKER_TAG -e GIT_COMMIT -e BUILD_NUMBER -v $PWD:/working -v sbt-cache:/root/.sbt -v jar-cache:/root/.ivy2 -v coursier-cache:/root/.cache/coursier hseeberger/scala-sbt:eclipse-temurin-17.0.2_1.6.2_2.13.8 /working/docker/clean_install.sh /working
```

Comment:
Rawls builds its jar via docker (why???), so we need to ensure the env vars get passed to the docker invocation.